From 1a70ce711c685c3292e10e5812d31f11dc6d41ad Mon Sep 17 00:00:00 2001
From: Antoine Belvire <antoine.belvire@opensuse.org>
Date: Fri, 15 Apr 2022 18:49:30 +0200
Subject: [PATCH] Fix incorrect parent of a control in FileViewer

This caused an assert with wxGTK 3.1.6, which checks for this kind of
mistakes.

Fixes #745
---
 src/fileviewer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fileviewer.cpp b/src/fileviewer.cpp
index 63d5a1336c..e8516e3f81 100644
--- a/src/fileviewer.cpp
+++ b/src/fileviewer.cpp
@@ -136,7 +136,7 @@ FileViewer::FileViewer(wxWindow*)
 #endif
     m_topBarSizer->Add(m_openInEditor, wxSizerFlags().Center().ReserveSpaceEvenIfHidden().Border(wxLEFT, PX(10)));
 
-    sizer->Add(new wxStaticLine(this, wxID_ANY, wxDefaultPosition, wxSize(-1, 1)), wxSizerFlags().Expand().Border(wxLEFT|wxRIGHT, PX(5)));
+    sizer->Add(new wxStaticLine(panel, wxID_ANY, wxDefaultPosition, wxSize(-1, 1)), wxSizerFlags().Expand().Border(wxLEFT|wxRIGHT, PX(5)));
 
     m_content = wxWebView::New(panel, wxID_ANY);
     sizer->Add(m_content, 1, wxEXPAND);
